device-types/HP/HP-Z420.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: HP
+model: Z420
+slug: hp-z420
+part_number: hp-z420
+u_height: 2
+is_full_depth: false  # 15.75"
+weight: 29.92
+weight_unit: lb
+front_image: true
+rear_image: true
+comments: '[Datasheet](http://www.hp.com/united-states/campaigns/workstations/images/datasheet_z420.pdf)'
+interfaces:
+  - name: ethernet-1/1
+    type: 1000base-t
+module-bays:
+  - name: PS1
```

Comment:
As the power is integrated into the device it needs to be added as a `power-port`.